app/views/manageBusinesses/manage/ManageObligations.scala.html

```diff
@@ -131,10 +131,8 @@
             )
         )
 
-        <h2 class="govuk-label-wrapper">
-            <label class="govuk-label govuk-label--l govuk-!-margin-top-8">
+        <h2 class="govuk-heading-m">
             @getUniqueMessage("heading")
```

Comment:
Can we do it so that we aren't changing what the old completion page heading looked like? It only needs to be smaller for the new one